### ISSUE TITLE
Prompt is slow when enabled

### ISSUE THREAD

**Issue by flands** [jonmosco/kube-ps1#59]
First off, awesome prompt! I enabled it and noticed that it appears to significantly slow down my prompt. The simple test is hitting enter a few times. If I run kubeoff the response is immediate, but if I run kubeon then hitting enter multiple times results in a rather significant delay. Unfortunately, I have not had much time to look into why this might be the case, but figured I would file an issue.

**Comment by jonmosco** [comment]
Thanks for your interest in the project!

Can you provide some benchmarks when it is enabled vs. disabled?  Which shell are you using?  Did you try it with the `--no-rc` options listed at the bottom of the README?

**Comment by flands** [comment]
Hey -- I am on a macbook pro and using BASH: GNU bash, version 4.4.19(1)-release (x86_64-apple-darwin17.3.0). I am running a simple test where I run the date command following by the enter key 10 times followed by the date command again.

* With kubeoff -g beforehand I see: 07/18/2018_12:53:25:944166000 to 07/18/2018_12:53:26:517969000 (~0.5 second)
* With kubeon -g beforehand I see: 07/18/2018_12:53:14:447028000 to 07/18/2018_12:53:18:814732000 (~4.5 seconds)

I did not see the `bash --norc` -- I tried this and get much better results: 07/18/2018_13:04:35:659182000 to 07/18/2018_13:04:36:340597000 (~0.7 second)

Any suggestions on how to go about troubleshooting this? :-/
